K8SSAND-1196 ⁃ Unable to build kind clusters locally due to missing tag on cass-operator

<!--

Feel free to ask questions on Discord at https://discord.com/invite/y4s64xC9 or in the forum at https://forum.k8ssandra.io/. 
-->

**What happened?**

While trying to run _make multi-up_ to test out the operator locally, I got the following error:

    for ((i = 0; i < 2; ++i)); do \
            kind delete cluster --name k8ssandra-$i; \
    done
    Deleting cluster "k8ssandra-0" ...
    Deleting cluster "k8ssandra-1" ...
    /home/ricky/devel/k8ssandra/k8ssandra-operator/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
    Error: err: exit status 1: stderr: go: github.com/k8ssandra/cass-operator@v1.8.0-rc.2.0.20220124145237-9d1c58a5dec6: invalid pseudo-version: preceding tag (v1.8.0-rc.2) not found
    
    Usage:
      controller-gen [flags]
    
    Examples:
            # Generate RBAC manifests and crds for all types under apis/,
            # outputting crds to /tmp/crds and everything else to stdout
            controller-gen rbac:roleName=<role name> crd paths=./apis/... output:crd:dir=/tmp/crds output:stdout
    
            # Generate deepcopy/runtime.Object implementations for a particular file
            controller-gen object paths=./apis/v1beta1/some_types.go
    
            # Generate OpenAPI v3 schemas for API packages and merge them into existing CRD manifests
            controller-gen schemapatch:manifests=./manifests output:dir=./manifests paths=./pkg/apis/... 
    
            # Run all the generators for a given project
            controller-gen paths=./apis/...
    
            # Explain the markers for generating CRDs, and their arguments
            controller-gen crd -ww
    
    
    Flags:
      -h, --detailed-help count   print out more detailed help
                                  (up to -hhh for the most detailed output, or -hhhh for json output)
          --help                  print out usage and a summary of options
          --version               show version
      -w, --which-markers count   print out all markers available with the requested generators
                                  (up to -www for the most detailed output, or -wwww for json output)
    
    
    Options
    
    
    generators
    
    +webhook                                                                                                                                                                   package  generates (partial) {Mutating,Validating}WebhookConfiguration objects.                        
    +schemapatch[:generateEmbeddedObjectMeta=<bool>],manifests=<string>[,maxDescLen=<int>]                                                                                     package  patches existing CRDs with new schemata.                                                      
    +rbac:roleName=<string>                                                                                                                                                    package  generates ClusterRole objects.                                                                
    +object[:headerFile=<string>][,year=<string>]                                                                                                                              package  generates code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.  
    +crd[:allowDangerousTypes=<bool>][,crdVersions=<[]string>][,generateEmbeddedObjectMeta=<bool>][,maxDescLen=<int>][,preserveUnknownFields=<bool>][,trivialVersions=<bool>]  package  generates CustomResourceDefinition objects.                                                   
    
    
    generic
    
    +paths=<[]string>  package  represents paths and go-style path patterns to use as package roots.  
    
    
    output rules (optionally as output:<generator>:...)
    
    +output:artifacts[:code=<string>],config=<string>  package  outputs artifacts to different locations, depending on whether they're package-associated or not.   
    +output:dir=<string>                               package  outputs each artifact to the given directory, regardless of if it's package-associated or not.      
    +output:none                                       package  skips outputting anything.                                                                          
    +output:stdout                                     package  outputs everything to standard-out, with no separation.                                             
    
    run `controller-gen object:headerFile=hack/boilerplate.go.txt paths=./... -w` to see all available markers, or `controller-gen object:headerFile=hack/boilerplate.go.txt paths=./... -h` for usage
    make: *** [Makefile:111: generate] Error 1It seems the 1.8.0-rc.2 tag is missing on the dependent cass-operator repo:

[https://github.com/k8ssandra/cass-operator/tags](https://github.com/k8ssandra/cass-operator/tags)

I tried to bump the version of cass-operator to the latest but then I get error:

```
[ricky@localhost k8ssandra-operator]$ git diff
diff --git a/go.mod b/go.mod
index e67b613..45ac642 100644
--- a/go.mod
+++ b/go.mod
@@ -16,7 +16,7 @@ require (
        github.com/golang/snappy v0.0.4 // indirect
        github.com/google/uuid v1.2.0
        github.com/gruntwork-io/terratest v0.37.7
-       github.com/k8ssandra/cass-operator v1.8.0-rc.2.0.20220124145237-9d1c58a5dec6
+       github.com/k8ssandra/cass-operator v1.9.0
        github.com/k8ssandra/reaper-client-go v0.3.1-0.20220114183114-6923e077c4f5
        github.com/pkg/errors v0.9.1
        github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.52.1
[ricky@localhost k8ssandra-operator]$ go mod tidy
go: finding module for package github.com/k8ssandra/cass-operator/apis/control/v1alpha1
github.com/k8ssandra/k8ssandra-operator imports
        github.com/k8ssandra/cass-operator/apis/control/v1alpha1: module github.com/k8ssandra/cass-operator@latest found (v1.9.0), but does not contain package github.com/k8ssandra/cass-operator/apis/control/v1alpha1
```

**Did you expect to see something different?**

The _make multi-up_  would deploy the operator on multiple kind clusters.

**How to reproduce it (as minimally and precisely as possible)**:

```
git clone https://github.com/k8ssandra/k8ssandra-operator
make multi-up
```

**Environment**

Fedora 34
kind v0.11.1 go1.16.13 linux/amd64
go version go1.16.13 linux/amd64

**K8ssandra Operator version**

Latest from main branch.
As of writing of this issue the commit SHA is 143afb17f65f8c52cbe14a6bfd1a516b36d7413a.

**Anything else we need to know?**:

This is my first issue on this repo, I'm happy to get further indications to push up a PR.

Thanks



┆Issue is synchronized with this [Jira Task](https://k8ssandra.atlassian.net/browse/K8SSAND-1196) by [Unito](https://www.unito.io)
┆friendlyId: K8SSAND-1196
┆priority: Medium

--- a/go.mod
+++ b/go.mod
@@ -16,7 +16,7 @@ require (
 	github.com/golang/snappy v0.0.4 // indirect
 	github.com/google/uuid v1.2.0
 	github.com/gruntwork-io/terratest v0.37.7
-	github.com/k8ssandra/cass-operator v1.8.0-rc.2.0.20220124145237-9d1c58a5dec6
+	github.com/k8ssandra/cass-operator v1.7.1-0.20220124145237-9d1c58a5dec6
 	github.com/k8ssandra/reaper-client-go v0.3.1-0.20220114183114-6923e077c4f5
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.52.1

